### PR TITLE
containers: add nfs server

### PR DESCRIPTION
**Pull request by pbrezina**
This PR adds nfs server in separate container that can be used to test autofs.

### CODE REVIEW

File: src/ansible/roles/nfs/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    name: '{{ item }}.service'
+    enabled: yes
+    state: started
+  with_items:
+  - nfs-server
+  - rpcbind
+
+# Due to limitations of container file system /exports must be mounted as a
+# volume and all shared directories must be created under /exports. This
+# is to make sure that the folder always exist.
```
Comment (ikerexxe) [inline src/ansible/roles/nfs/tasks/main.yml:12]: s/exist/exists

File: src/docker-compose.build.yml [hunk 2]
```diff
@@ -14,3 +14,6 @@ services:
   client:
     image: localhost/sssd/ci-base-client:${TAG}
     container_name: sssd-wip-client
+  nfs:
+    image: localhost/sssd/ci-base-nfs:${TAG}
+    container_name: sssd-wip-nfs
```
Comment (ikerexxe) [inline src/docker-compose.build.yml:19]: Missing EOF

File: data/ssh-keys/hosts/nfs.ecdsa_key [hunk 3]
```diff
@@ -0,0 +1,9 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```
Comment (ikerexxe) [inline data/ssh-keys/hosts/nfs.ecdsa_key:1]: I guess that adding the private key is intentional, right?

**Review by ikerexxe — COMMENTED** [review]
I added some minor comments. On top of that, can you update the [flowchart](https://github.com/SSSD/sssd-ci-containers/blame/master/readme.md#L272) with the new image?

File: data/ssh-keys/hosts/nfs.ecdsa_key (hunk 3 above)
Comment (pbrezina) [reply data/ssh-keys/hosts/nfs.ecdsa_key:1]: Yes.

File: src/docker-compose.build.yml (hunk 2 above)
Comment (pbrezina) [reply src/docker-compose.build.yml:19]: Fixed.

**Comment by pbrezina** [comment]
Done. I also workaround the need for /exports directory so I removed it. (Tests now use /dev/shm/exports which is always tmpfs in the containers so can hold nfs shares).

**Review by ikerexxe — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**